### PR TITLE
Update node an proxy 

### PR DESCRIPTION
**Pull request by miiu96**
- Node to version 1.1.37
- Proxy to version 1.1.9

### CODE REVIEW

File: mainnet/configs/config.toml [hunk 1]
```diff
@@ -3,27 +3,37 @@
    # ServerPort is the port used for the web server. The frontend will connect to this port
    ServerPort = 8079
 
-   # RequestTimeoutSec represents the maximum number of seconds a request can last until throwing an error
-   RequestTimeoutSec = 10
+    # RequestTimeoutSec represents the maximum number of seconds a request can last until throwing an error
```
[the remark this answers is not in the document]
Comment (miiu96) [reply mainnet/configs/config.toml:6]: done.

File: mainnet/configs/config.toml [hunk 2]
```diff
@@ -3,27 +3,37 @@
...
+    BalancedFullHistoryNodes = true
+
+    # FaucetValue represents the default value for a faucet transaction. If set to "0", the faucet feature will be disabled
+    FaucetValue = "0"
+
+    # RateLimitWindowsDurationSeconds represents the time window for limiting the number of requests to a given API endpoint
+    # For example, if RateLimitDurationSeconds = 60 and the endpoint /address/:address/nonce is rate-limited to 5,
+    # then after 5 requests in a 60 seconds window, a 'Too many requests' response will be returned. There is a clean-up
+    # mechanism so after RateLimitDurationSeconds seconds, the restrictions will be reset.
+    RateLimitWindowDurationSeconds = 60
```
[the remark this answers is not in the document]
Comment (miiu96) [reply mainnet/configs/config.toml:36]: done.

**Review by bogdan-rosianu — APPROVED** [review]

File: mainnet/.env [hunk 3]
```diff
@@ -18,5 +18,5 @@ IP_1=10.0.0.5
 IP_2=10.0.0.4
 IP_M=10.0.0.3
 
-NODE_TAG=elrond-node-obs:v1.1.25
-PROXY_TAG=elrond-proxy:v1.1.4
\ No newline at end of file
+NODE_TAG=elrond-node-obs:v1.1.37
+PROXY_TAG=elrond-proxy:v1.1.9
```
Comment (iulianpascalau) [inline mainnet/.env:22]: please add empty line

**Review by iulianpascalau — APPROVED** [review]

File: mainnet/.env (hunk 3 above)
Comment (miiu96) [reply mainnet/.env]: done.

**Pull request merged**